rust-install/src/manifest/mod.rs

```diff
@@ -8,6 +10,14 @@ use install::InstallPrefix;
 use openssl::crypto::hash::{Type, Hasher};
 use itertools::Itertools;
 
+pub const DIST_MANIFEST: &'static str = "multirust-dist.toml";
```

Comment:
This should probably be named like the channel manifest, maybe `multirust-channel-manifest.toml`.